enginetest/queries.go

```diff
@@ -5395,7 +5395,8 @@ var InfoSchemaQueries = []QueryTest{
 				"  `s` varchar(20) NOT NULL COMMENT 'column s',\n" +
 				"  PRIMARY KEY (`i`),\n" +
 				"  KEY `mytable_i_s` (`i`,`s`),\n" +
-				"  UNIQUE KEY `mytable_s` (`s`)\n" +
+				"  UNIQUE KEY `mytable_s` (`s`),\n" +
+				"  CONSTRAINT `mycheck` CHECK (`i` > -100)\n" +
```

Comment:
You actually can't do this to `myTable`, it's just too widely used. With this setup, nobody gets to use myTable unless they implement check constraints

Instead, conditionally create a new table in test_data.go only if the harness supports checks, OR

Just test this in the existing Check methods in enginetests.go (which integrators can decide to not test if they can't support it)